test/byoc.test.ts

```diff
@@ -168,6 +168,29 @@ describe("BYOC", () => {
     // validate that reading it twice doesn't create resources twice
     const _ = byoc.targetGroups.admin.application;
   });
+
+  test("With otel collector", () => {
+    const { stack, vpc } = createStack();
+
+    new RestateBYOC(stack, "with-otel-collector", {
```

Comment:
Minor but do you want to add this to the snapshots? Might be neat having the CFn output checked in, that way future changes will also show up in the snapshot diffs.

Reply:
i wonder if i should just enable otel in the main snapshot, and then have this one just check we dont throw with it disabled. i hate having combinatorial snapshots they get huge